scripts/cibuild.sh

```diff
@@ -21,6 +21,6 @@ then
     else
         ./scripts/update.sh
         ./scripts/test.sh
-        npm run bundle
+        ./src/app/bundle.sh --vendor --minify
```

Comment:
I think the bundle script should be called from `update`. Maybe we can use a "production" environment variable to indicate that minification needs to happen.